xivo/tests/test_anysql.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022-2022 The Wazo Authors  (see the AUTHORS file)
```

Comment:
```suggestion
# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
```

Reply:
It's really hard to know which old libraries will still be used. I'm sure that would be a much better option. I was just doing a quick test to see if agid would run under python3 and noticed that this made it fail on import. It was such a simple fix I thought I'd do it, but if we're re-writing agid and not just porting it then I guess it may not be useful.

Reply:
No worries, I didn't see the agid PR related
My only point is that python3 migration should be discussed before merging something.
No issue to make tests and exploration :)

Reply:
Oh for sure! Definitely wasn't planning on merging agid. I just saw that this library was supposed to support python3, but would fail if imported.